### ISSUE TITLE
Bitrate setting

### ISSUE THREAD

**Issue by Encryptize** [azlux/botamusique#62]
**Is your feature request related to a problem? Please describe.**
I want to get a setting for set a bitrate for a bot without change settings of Mumble server.

**Comment by Mati20187** [comment]
I'd also really appreciate that feature

**Comment by azlux** [comment]
It's currently not supported by pymumble.
The librairy use bitrate from server configuration.

**Comment by azlux** [comment]
I reopen this old issue because after read pymumble code, there are already a bandwidth function : https://github.com/azlux/pymumble/blob/pymumble_py3/pymumble_py3/mumble.py#L440

So I just need to implement the setting into botamusique

**Comment by felix91gr** [comment]
@azlux so if I understood correctly, botamusique currently picks the largest bitrate allowed by the server?

**Comment by TerryGeng** [comment]
@felix91gr Yes. This bitrate is selected by pymumble. https://github.com/azlux/pymumble/blob/a6f4952c105065be1fe57e8bfadfbc5dbb48bcf8/pymumble_py3/mumble.py#L325

**Comment by felix91gr** [comment]
We were hearing some compression artifacts in the music (probably opus related), so we went ahead and raised our bandwidth cap in the `murmur.ini` file, from 72kbps to 260kbps. That seemed to do the trick. We tried 130 before and still heard artifacts, so we just duplicated it to 260, which worked perfectly. I'm guessing that maybe around 200 would be enough, though.

What is your bandwidth setting, @TerryGeng? Have you raised it from the default?
